ci/citest/acceptance-test/tests/local_image_test.go

```diff
@@ -0,0 +1,27 @@
+// +build acceptance
+
+package tests
+
+import (
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestLocalImage(t *testing.T) {
+
+	// Use custom lxc-template.
+	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc", `{"lxc_template":{"template":"openvdc"}}`)
```

Comment:
Here is missing ``node_groups`` attribute to let the instance run on ``executor_lxc_ip`` 10.100.0.14.

```
`{"lxc_template":{"template":"openvdc"}, "node_groups":"linuxbr"}`
```